### PR TITLE
Version 2.4.0. New Brawler Rosa! New skins! Improved Trophy Road! Balance changes!

### PR DESCRIPTION
**Pull request by lenndevv**

### CODE REVIEW

File: brawlcord/utils/emojis.py [hunk 1]
```diff
@@ -239,8 +243,8 @@
     "Bronze": "<:l2:645337384447377409>",
     "Silver": "<:l3:645337384657092638>",
     "Gold": "<:l4:645337384174616577>",
-    "Crystal": "<:l5:645337385500016640>",
-    "Diamond": "<:l6:645337387152441375>",
+    "Crystal": "<:l6:645337387152441375>",
```
Comment (lenndevv) [inline brawlcord/utils/emojis.py:246]: Fixed a bug where the league icons are incorrect

File: brawlcord/data/trophy_road.json [hunk 2]
```diff
@@ -560,7 +560,7 @@
     "94": {
         "Trophies": 14500,
         "RewardType": 1,
-        "RewardCount": 150,
+        "RewardCount": 1000,
         "RewardExtraData": ""
```
Comment (lenndevv) [inline brawlcord/data/trophy_road.json:564]: Improved the trophy road. Thinking of adding a compensation for players who already progressed a lot in the TR, might extend the trophy road after this update

File: brawlcord/utils/brawlers.py [hunk 3]
```diff
@@ -575,6 +575,17 @@ def super_info(self, stats):
...
+class Rosa(Brawler):
+    """Class to represent Rosa."""
+
+    def super_info(self, stats):
+        super_str = self.ult['extra']
+        super_desc = f"```{self.ult['desc']}```"
+
+        info = f"{super_desc}\n{emojis['super']} {super_str}: {self.ult['duration']} seconds"
+
+        return info
```
Comment (lenndevv) [inline brawlcord/utils/brawlers.py:589]: Rosa's Super mechanics are identical to Leon's. This is purely a placeholder, I don't know how to impliment a damage reduction mechanic

Comment (lenndevv) [reply brawlcord/utils/brawlers.py:589]: Oops, accidentally made a review. Just pretend it's a comment ;)

**Comment by snowsee** [comment]
Closed in favour of separate PRs for individual changes.

**Pull request closed without merging**